packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -1323,6 +1332,33 @@ export function renderDidSuspend(): void {
   }
 }
 
+export function renderDidSuspendUncaptured(): void {
```

Comment:
I copied the checks from `renderDidSuspendDelayIfPossible` but I don't fully understand them